app/views/users/index.html.haml

```diff
@@ -14,7 +14,6 @@
 
   - if @users.any?
     = render "search/results_table",
```

Comment:
Should we dig into this view and disable the order for link when you're in cross facility mode?

Reply:
I'm not sure - I thought Jeff said they were going to think on it.  Should be an easy change to make if the want it done.  Should I check in with them about it?

Reply:
You're right that it's probably an easy change so we can wait for them to give us the go ahead to change that.